lib/server.js

```diff
@@ -43,13 +43,14 @@ module.exports = {
   registerRoutes (app, server) {
     _.each(app.routes, route => server.route(route))
 
-    if (app.config.web.assets) {
+    if (app.config.main.paths.www) {
```

Comment:
I didn't say you have to add it but if you choose to don't I will remove it from express to have same behaviour :) open discussion here let me know